scripts/flattener.py

```diff
@@ -1195,9 +1175,9 @@ def main(mfinal_id):
 		if 'Full list available in logging file.' not in n:
 			logging.warning(n)
 
-args = getArgs()
-connection = lattice.Connection(args.mode)
-server = connection.server
 
 if __name__ == '__main__':
-    main(args.file)
+	args = getArgs()
+	connection = lattice.Connection(args.mode)
+	server = connection.server
```

Comment:
I've wondered about this line before. Is `server` being actually used anywhere?

Reply:
Not anywhere in the flattener. Git blame shows this was present since the very first flattener version, so probably just copied over from other scripts that needed to more explicitly use the `Connection.server` attribute. This line could be deleted in the flattener

`Connection.server` is the attribute to hold the environmental variable `{PROD/DEMO}_SERVER`

Some of the other scrips assign the server attribute to the `server` variable, but even then,
`connection.server` would probably be fine.

Reply:
Sounds good! Let's go ahead and delete the line from the flattener. Thanks!

Reply:
Deleted the line and quickly tested locally with `test_flattener.py` defaults and a normal flattener run with LATDF190KNY

Reply:
Perfect! Thank you!